Hamburg/HLC/RDW006.xml

```diff
@@ -0,0 +1,410 @@
+                              <note>Number of characters per line: 7-10</note>
+                              <dimensions unit="mm" xml:id="textarea3">
+                                 <height>480</height>
+                                 <width>65</width>
+                              </dimensions>
+                           </layout>
+                        </layoutDesc>
+                  </objectDesc>
+                  
+                      <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                            <date notBefore="1800"/>
+                           <desc>Mediocre hand, irregular.</desc>
+                           <seg type="ink">Black, pinkish red.</seg>
+                           <seg type="script">Modern bulky script.</seg>
+                           <seg type="rubrication">Beginnings of texts, the word <foreign xml:lang="gez">እግዚአብሔር</foreign>, single words.</seg>
+                        </handNote>
+                     </handDesc>
+                  
+                  <decoDesc>
+                     <decoNote xml:id="d1" type="drawing" corresp="#pic1">
+                        <desc>Multichrome drawing of an <term key="angel"/> holding a <term key="sword"/> and a <term key="scabbard"/>,
+                           between ornamental bands.</desc>
+                        <dimensions unit="mm">
+                           <height>170</height>
+                           <width>78</width>
+                        </dimensions>
+                     </decoNote>
+                     
+                     <decoNote xml:id="d2" type="drawing" corresp="#pic2">
+                        <desc>Multichrome drawing of two <term key="angel">angels</term> between ornamental bands.</desc>
+                        <dimensions>
+                           <height>70</height>
+                           <width>70</width>
+                        </dimensions>
+                     </decoNote>
+                     
+                     <decoNote xml:id="d3" type="drawing" corresp="#pic3">
+                        <desc>Multichrome drawing of magic squares, every second square containing the design of a human face.</desc>
+                        <dimensions>
+                           <height>100</height>
+                           <width>78</width>
+                        </dimensions>
+                     </decoNote>
+                     
+                     <decoNote xml:id="d4" type="drawing" corresp="#pic4">
+                        <desc>Multichrome drawing of a human face between geometric designs.</desc>
+                        <dimensions>
+                           <height>85</height>
+                           <width>76</width>
+                        </dimensions>
+                     </decoNote>
+                  </decoDesc>
+
+               </physDesc>
+
+               <history>
+                  <origin>
+                     <origDate notAfter="1800"/>
```

Comment:
notBefore?

Reply:
You're right, I was hoping I could make it pass like this...I can just write 2019, hoping we'll get information on the acquisition.